### ISSUE TITLE
clean up giant for loop in ffcompare.py

### ISSUE THREAD

**Issue by bannanc** [MobleyLab/off-ffcompare#32]
This for loop is REALLY hard to follow as someone who didn't write it... I'm going to try to either split it up, or propose some suggestions. I'll open a pull request soon.

**Comment by bannanc** [comment]
I started this in pull request #33 but the work is on going

**Comment by vtlim** [comment]
@bannanc - good to close?

**Comment by bannanc** [comment]
I think it is as clean as it is going to get.
